Group Quota SDK

tejasm-microsoft opened the pull request:
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

tejasm-microsoft committed:
first commit for draft PR

tejasm-microsoft committed:
first commit for draft PR

jsquire reviewed (changes requested):
@tejasm-microsoft :

Your account lacks the required public GitHub organizations and permissions required of an internal contributor.  Please review the Azure SDK onboarding documentation and use the associated Teams channel for support.
- [Azure SDK onboarding](https://eng.ms/docs/products/azure-developer-experience/onboard/requirements) _(Microsoft internal)_
- [Azure SDK onboarding assistance](https://teams.microsoft.com/l/channel/19%3ade3fce22c2994be18cac50502c55f717%40thread.skype/Azure%2520SDK%2520Onboarding?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) _(Microsoft internal)_

You can verify the state of your account by running the [Validate-AzsdkCodeOwner](https://github.com/Azure/azure-sdk-tools/blob/main/tools/github/scripts/Validate-AzsdkCodeOwner.ps1) script from the Azure SDK tools repository.

Please also be sure to add yourself to [CODEOWNERS](https://github.com/Azure/azure-sdk-for-net/blob/main/.github/CODEOWNERS) for this library, if you will be maintaining it going forward.

tejasm-microsoft committed:
adds setGroupQuotaTest

tejasm-microsoft committed:
adds setGroupQuotaTest

azure-sdk commented:
**API change check**

API changes are not detected in this pull request.

jsquire commented on sdk/core/Azure.Core/src/Shared/NextLinkOperationImplementation.cs:
```diff
@@ -532,8 +532,8 @@ private static bool IsFinalState(Response response, HeaderSource headerSource, o
                             case HeaderSource.None when root.TryGetProperty("properties", out var properties) && properties.TryGetProperty("provisioningState", out JsonElement property):
-                            case HeaderSource.OperationLocation when root.TryGetProperty("status", out property):
-                            case HeaderSource.AzureAsyncOperation when root.TryGetProperty("status", out property):
+                            case HeaderSource.OperationLocation when root.TryGetProperty("properties", out properties) && properties.TryGetProperty("provisioningState", out property):
```
This needs to be reverted.   You should not be making changes to Azure Core files here.

tejasm-microsoft replied:
Reverted the changes made in the core files

tejasm-microsoft committed:
test updates

tejasm-microsoft committed:
test updates

allchiang-msft committed:
Update ACS rooms test resource directory path in tests.yml (#44660)

rahuls-microsoft commented:
/azp run

tejasm-microsoft commented:
/azp run

tejasm-microsoft committed:
revert erroneous core library change

tejasm-microsoft committed:
revert erroneous core library change

azure-sdk committed:
Update AutoRest C# version to 3.0.0-beta.20240619.1 (#44668)

Commit:
[AzureMonitor] upgrade OTel to 1.9.0 (#44650)

* upgrade OTel to 1.9.0

* changelog

JoshLove-msft committed:
SystemEvents beta (#44659)

* SystemEvents beta

* CI

* remove samples

* Update service.projects

* Fix description

* Update sdk/eventgrid/Azure.Messaging.EventGrid.SystemEvents/README.md

Co-authored-by: Jesse Squire <jesse.squire@gmail.com>

---------

Co-authored-by: Jesse Squire <jesse.squire@gmail.com>

JoshLove-msft committed:
Internalize SystemEventsClient (#44677)

jsquire committed:
[Service Bus] Message copy test fix (#44655)

The focus of these changes is to adjust test expectations
to account for the scheduled enqueue time being
cleared when a copy is made.

danieljurek committed:
Add SYSTEM_ACCESSTOKEN (#44678)

tejasm-microsoft committed:
remove unneccessary added feed

tejasm-microsoft committed:
remove unneccessary added feed

Commit:
[AzureMonitor] Integration Test with two Application Insights resource (part 1) (#44634)

* refactor bicep

* added second resource

* testing unneeded elements

* final update to bicep and Environment file

* refactor test to validate both resources

* pr feedback. rename primary resources. remove 2nd resource test

* Revert "pr feedback. rename primary resources. remove 2nd resource test"

This reverts commit 06cd60e1f2f8f8f1e77af33e847462d1805013f7.

* remove second test. we spoke offline this isn't needed

* update variable names. this was pr feedback.

Commit:
[AzureMonitorDistro] sync vendored code (post 1.9.0) (#44682)

* update sql

* copy resources.azure

* update shared

* fix namespace

* changelog

jorgerangel-msft committed:
[AppConfiguration] add support for filtering by tags (#44387)

* feat: add support for filtering by tags

* address initial feedback

* address additional feedback

* update sample

* update test recordings

* spacing change

* generate api and sample files

danieljurek committed:
Set AZURESUBSCRIPTION_CLIENT_ID and AZURESUBSCRIPTION_TENANT_ID (#44684)

Commit:
[AzureMonitor] Integration Test with two Application Insights resource (part 2) (#44656)

* refactor bicep

* added second resource

* testing unneeded elements

* final update to bicep and Environment file

* refactor test to validate both resources

* pr feedback. rename primary resources. remove 2nd resource test

* saving work in progress. working example using Exporter x2

* refactor tests and merge classes

* Revert "pr feedback. rename primary resources. remove 2nd resource test"

This reverts commit 06cd60e1f2f8f8f1e77af33e847462d1805013f7.

* remove second test. we spoke offline this isn't needed

* update variable names. this was pr feedback.

* add test with Distro and update recording

* pr feedback. update service variables. re-record.

amariwest-msft committed:
Move CallAutomation GA3 Changes to ALPHA3 (#44307)

* init swagger gen w/ latest preview version

* add manual BYOS features

* added GA3 features with alpha3 swagger

* init swagger gen w/ latest preview version

* add manual BYOS features

* added GA3 features with alpha3 swagger

* fix for failing tests

* fix for failing tests

* fix for failing tests

* fix for failing tests

* fix for failing tests

* ran export api

* comment out failing assert

* comment out failing assert

* corrected asserts

* corrected CHANGELOG

* corrected CHANGELOG

* fixed type issue with test

* updated CHANGELOG.md

* updated CHANGELOG.md

* removed added api files

* reverted changelog

ahsonkhan committed:
Add back-ticks in AzurePipelinesCredential section of the Identity TSG (#44664)

azure-sdk committed:
Increment package version after release of Azure.Messaging.EventGrid.SystemEvents (#44683)

azure-sdk committed:
Sync eng/common directory with azure-sdk-tools for PR 8475 (#44658)

* update the target version of the proxy

* use the new version of the proxy. this is a breaking change for a few container registry tests which I need to re-record

* add re-recorded tests!

---------

Co-authored-by: Scott Beddall (from Dev Box) <scbedd@microsoft.com>

azure-sdk committed:
Force amd64 as the build platform - it has to match what the AKS cluster is running on. (#44688)

Co-authored-by: Richard Park <ripark\@microsoft.com>

chunyu3 committed:
Resolve openai failure (#44700)

* adopt OpenAI 2.0.0-beta.6

* update openAI readme

bewatersmsft committed:
Azure Firewall Save and Commit UTs (#44597)

azure-sdk committed:
Increment package version after release of Azure.ResourceManager.Network (#44703)

benbp committed:
Test subnet mapping (#44657)

ralph-msft committed:
Add fine tuning tests (#44695)

- Fixes some issues with the FineTuningClient and FileClient that were preventing fine tuning scenarios from working
  - Wrong paths were being used in the generated REST requests
  - Multi-part uploads were not setting the required Content-Type header correctly for each the file part
  - File uploads for fine tuning return HTTP 201 which was treated as an error before
- Adds more thorough fine tuning model tests, as well as all the required types and clients to be able exercise all parts of the fine tuning model scenario

Commit:
[AzureMonitor] update internals in test project (#44713)

* saving work in progress. Net tests are passing

* delete extra file

* testing collection definition

* cleanup

* cleanup

ArthurMa1978 committed:
Add `Features Added` section for first beta (#44628)

azure-sdk committed:
Update AutoRest C# version to 3.0.0-beta.20240624.1 (#44714)

* Update Generator Version 3.0.0-beta.20240624.1

* Update SDK codes de_he_2

* Update SDK codes ad_co_0

* Update SDK codes st_wo_6

* Update SDK codes mi_pu_4

* Update SDK codes co_de_1

* Update SDK codes he_me_3

* update changelog

---------

Co-authored-by: FAREAST\chunyu <chunyu@microsoft.com>

chunyu3 committed:
Adopt tsp-client and adjust script steps (#44689)

* adopt tsp-client

* adopt tsp-client for ci pipeline

* remove autorest.md for typespec project

* change the pack location

* search artifact in Release directory

* remove commented code

* remove typespec generation scripts

* Revert "remove typespec generation scripts"

This reverts commit 59c813215a64dfbed6fa2e699c6cfa85b783725b.

* use servce.proj instead of solution file

* use local spec

* Update eng/scripts/automation/GenerateAndBuildLib.ps1

ArthurMa1978 committed:
Prepare release for Oracle first beta (#44720)

EmmaZhu committed:
Remove blob type from BlobCheckpointData (#44701)

* Remove blob type from BlobCheckpointData

* Remove an accidential added reference.

azure-sdk committed:
Update AutoRest C# version to 3.0.0-beta.20240625.2 (#44722)

* Update Generator Version 3.0.0-beta.20240625.2

* Update SDK codes de_he_2

* Update SDK codes ad_co_0

* Update SDK codes st_wo_6

* Update SDK codes mi_pu_4

* Update SDK codes pu_sq_5

* Update SDK codes he_me_3

* Update SDK codes co_de_1

* fix issues

---------

Co-authored-by: Arcturus Zhang <dapzhang@microsoft.com>

pjohari-ms committed:
[Microsoft.DocumentDB] CosmosDB - New API version 20240515 (#44635)

* Generated Models and samples

* Service bugfix generation and test changes

* API Compat changes due to bugfix

* Added test recordings

* update

* update

---------

Co-authored-by: Arthur Ma <arthurma@microsoft.com>

Commit:
[AzureMonitorDistro] Tests for Sql Instrumentation (#44697)

* saving work in progress. Net tests are passing

* delete extra file

* testing collection definition

* cleanup

* cleanup

* cleanup

* cleanup

* fix typo

* pr feedback

* use OTEL copyright

danieljurek committed:
Wire up SubscriptionConfigurationFilePaths (#44726)

yifanz7 committed:
[StorageMover] Update to API version 2024-07-01: Support upload limit schedule (#44551)

v-dharmarajv committed:
Updating the alpha version (#44730)

* updating the version for the alpha

* updating the alpha version

azure-sdk committed:
Update AutoRest C# version to 3.0.0-beta.20240625.3 (#44724)

* Update Generator Version 3.0.0-beta.20240625.3

* Update SDK codes co_de_1

azure-sdk committed:
Increment package version after release of Azure.ResourceManager.OracleDatabase (#44731)

azure-sdk committed:
Increment package version after release of Azure.ResourceManager.CosmosDB (#44732)

azure-sdk committed:
Increment package version after release of Azure.ResourceManager.StorageMover (#44733)

ArthurMa1978 committed:
Prepare for releasing SelfHelp 1.1.0-beta.4 (#44736)

azure-sdk committed:
Update Generator Version 3.0.0-beta.20240625.4 (#44737)

azure-sdk committed:
Increment package version after release of Azure.ResourceManager.SelfHelp (#44738)

mwesigwaguma committed:
Update sdk to use GA api version 2024-04 (#44692)

amnguye committed:
[Storage] [DataMovement] Added support for Share File to/from Blob for Single Transfer (#44712)

* WIP - updated E2E single transfer tests to include properties test

* Added Share File to Blob (vice versa) transfers properties and tests

* PR comments

Commit:
Adding AzDO pipeline authentication to TestEnvironment.Credential (#44707)

* Adding AzDO pipeline authentication to TestEnvironment.Credential

* temp changes for testing

* fix typo

* removed test hack

* fb

* inline pwsh fix

ralph-msft committed:
Adds test recording and playback for Azure OpenAI (#44711)

- Simplifies the base test class
- Simplifies the configuration
- Fixes race condition on some test case exit with the test proxy
- Adds recordings for all tests (or marks them as LiveOnly)

azure-sdk committed:
Increment package version after release of Azure.ResourceManager.ServiceFabricManagedClusters (#44741)

anamikapan committed:
Codeowners for resource manager path (#44581)

* change codeowners

* name change

* write permissions fix

* Update .github/CODEOWNERS

Co-authored-by: Wes Haggard <weshaggard@users.noreply.github.com>

* Update .github/CODEOWNERS

Co-authored-by: Wes Haggard <weshaggard@users.noreply.github.com>

---------

Co-authored-by: Wes Haggard <weshaggard@users.noreply.github.com>

ralph-msft committed:
Add recording for additional test case, automate recorded config capture (#44743)

* Updated the test fine tuning model deployment client to also enable capturing a recording and playing it back. That test case now also has a recording
* The playback configuration is now also captured and recorded while recording mode, and then automatically sanitized and saved saved back at the end of the recording to ensure it remains up to date automatically
* In playback mode the tests will now always use the saved "playback" configuration. This makes the tests running locally behave the same as those running in CI/CD to make it easier to catch issues

rajkumar-rangaraj committed:
[AzureMonitorDistro] Remove logs sampling. (#44745)

* Remove logs sampling.

* changelog update

* pr feedback - changelog

azure-sdk committed:
Update AutoRest C# version to 3.0.0-beta.20240627.1 (#44758)

* Update Generator Version 3.0.0-beta.20240627.1

* Update SDK codes de_he_2

* Update SDK codes mi_pu_4

* Update SDK codes he_me_3

* Update SDK codes co_de_1

* update api for healthinsights

---------

Co-authored-by: Arcturus Zhang <dapzhang@microsoft.com>

v-jamcheung committed:
Stable .NET SDK for Oracle.Database (#44681)

azure-sdk committed:
Update AutoRest C# version to 3.0.0-beta.20240627.2 (#44761)

* Update Generator Version 3.0.0-beta.20240627.2

* Update SDK codes pu_sq_5

* Update SDK codes co_de_1

* fix issues

---------

Co-authored-by: Arcturus Zhang <dapzhang@microsoft.com>

audunn committed:
[NetAppFiles] update to 2023-11-01 (#44274)

danieljurek committed:
Only add SubscriptionConfigurationFilePaths if UseFederatedAuth is true (#44742)

* Only add SubscriptionConfigurationFilePaths if UseFederatedAuth is true

* Fix up custom sub config in identity tests

JimSuplizio committed:
Fix namespaces generation to handle framework targets other than netstandard2.0 (#44768)

* Fix namespaces generation to handle framework targets other than netstandard2.0

* fix variable name

* If more than 1 is found, just use the first one

azure-sdk committed:
Increment package version after release of Azure.ResourceManager.NetApp (#44767)

benbp committed:
Add pool env variable to build-test-resource-config template (#44744)

azure-sdk committed:
Update AutoRest C# version to 3.0.0-beta.20240627.4 (#44772)

* Update Generator Version 3.0.0-beta.20240627.4

* Update SDK codes co_de_1

rajkumar-rangaraj committed:
[AzureMonitorExporter] Fix DuplicateKeyException when LogRecord contains EventId or EventName in both LogRecord and Attributes (#44748)

* Fix DuplicateKeyException when LogRecord

* Update changelog

* Update changelog

Commit:
[AzureMonitorExporter] add default value for SDK version string (#44747)

* add default value for SDK version string

* pr feedback

vicancy committed:
Release the official release for Web PubSub isolated function extension (#44756)

* Release the official release

* update readme

azure-sdk committed:
Update Generator Version 3.0.0-beta.20240627.5 (#44783)

pshao25 committed:
Remove some suppression of AZC0006 and AZC0007 (#44643)

blueww committed:
[Storage] Support SRP new API version 2023-05-01 (#44571)

Juliehzl committed:
Release 2024-03-01 for container apps (#44721)

almasak committed:
Informaticadatamanagement dot net SDK beta (#44624)

azure-sdk committed:
Increment package version after release of Azure.ResourceManager.InformaticaDataManagement (#44787)

azure-sdk committed:
Increment package version after release of Azure.ResourceManager.AppContainers (#44786)

azure-sdk committed:
Increment package version after release of Azure.ResourceManager.Storage (#44785)

scottaddie committed:
Edit pass on Azure OpenAI README (#44776)

* Edit pass on Azure OpenAI README

* More edits

ralph-msft committed:
Improve test playback (#44795)

* Reduces test recording size
* Shaves off 25-30 seconds of playback runtime for tests
* Update prompt for image generation to not accidentally trigger content filtering

hallipr committed:
Use new symbols publishing api (#44727)

* Use new symbols publishing api
* Update build-tools repo reference

JoshLove-msft committed:
Add TryGetSystemEventData (#44753)

* Add TryGetSystemEventData

* Delete separate samples folder

* regen

* docsettings

* remove broken link

* pr fb

* Rename

ArthurMa1978 committed:
Prepare for release Oracle first stable (#44809)

jsquire committed:
[CODEOWNERS] Fix linter issues (#44821)

The focus of these changes is to fix linter issues and make some overdue updates.

alexwiese committed:
Add overloads to send email using operation ID passed in from consumer (#40779)

azure-sdk committed:
Sync eng/common directory with azure-sdk-tools for PR 8371 (#44371)

* Restrict live test storage account access to client IP

* Add storage pools to test resources vnet allowlist

* Use pool subnet map to reduce number of subnets added to live test resources

* snap

* Add test resource parameter to add ip ranges to storage firewall

* Validate and update ip firewall rules

* Get pool subnet using separate workload identity

---------

Co-authored-by: Ben Broderick Phillips <bebroder@microsoft.com>

hallipr committed:
Update mgmt-release with new symbols yaml (#44823)

m-redding committed:
[Schema Registry] Custom & JSON schema GA (#44472)

* initial commit

* lru cache should be internal

* feedback

Commit:
refactor event id in demo and tests (#44797)

azure-sdk committed:
Update Generator Version 3.0.0-beta.20240701.1 (#44820)

ArcturusZhang committed:
update API for `Azure.Core.Experimental` (#44830)

azure-sdk committed:
Update AutoRest C# version to 3.0.0-beta.20240701.2 (#44832)

* Update Generator Version 3.0.0-beta.20240701.2

* Update SDK codes de_he_2

* Update SDK codes ad_co_0

* Update SDK codes mi_pu_4

* Update SDK codes pu_sq_5

* Update SDK codes he_me_3

* Update SDK codes co_de_1

* mitigate breaking changes

---------

Co-authored-by: Wei Hu <huwe@microsoft.com>

wiboris committed:
Azure.Compute.Batch inital checkin (#44524)

* inital checkin of azure.compute.batch

---------

Co-authored-by: Jesse Squire <jsquire@microsoft.com>

JialinXin committed:
Fix SetState() not work (#44819)

* Fix SetState not work

* Fix namespace.

* Add API updates.

zhangjiale-64 committed:
Fix spelling issues in all subpackages READMEs (#44844)

azure-sdk committed:
Increment package version after release of Azure.Compute.Batch (#44838)

amnguye committed:
Fixing constant in BlobSourceCheckpointData to have correct schema size (#44839)

amnguye committed:
[Storage] [DataMovement] [Shares] Preserve Permission Key (#44696)

* WIP

* Preserve Single Transfer Permissions; Rerecorded failing tests

* Cleanup

* Preserve permission key defaults to false

* PR Comments

* Removed unnecessary ResourceProperties in ShareDirectoryResource; Fixed LastWritten values; preserve permissions

* Rerecord tests

* Revert assests.json to main

* Rerecord tests

* Rerecord test to hopefully resolve merge conflict assets tag

* Added end to end tests for short permission values

azure-sdk committed:
Sync eng/common directory with azure-sdk-tools for PR 8528 (#44826)

* Fix default value for env vars in build-test-resource-config template

* Add empty pool condition

---------

Co-authored-by: Ben Broderick Phillips <bebroder@microsoft.com>

hakimms committed:
Updated entra auth events readme (#44178)

* Updated readme

* Updated changelog.md

* feedback changes and updated links

* Updated links and feedback suggestions

* Updated version and lowercase ref link

* Feedback changes

* Copied learn documentation to readme

* Added required titles to readme

* Updated structure

* removed locale from urls

* Feedback changes

* Added required section

* Feedback updates

* Updated layout to match other readme files

azure-sdk committed:
Sync eng/common directory with azure-sdk-tools for PR 8516 (#44799)

* Ensure subConfigFiles is not an empty string

* Skip instances where $file is an empty string

---------

Co-authored-by: Daniel Jurek <djurek@microsoft.com>

azure-sdk committed:
Sync eng/common directory with azure-sdk-tools for PR 8549 (#44841)

* commit the file changes so that we can see them running

* use standalone tool

---------

Co-authored-by: Scott Beddall (from Dev Box) <scbedd@microsoft.com>

azure-sdk committed:
Update AutoRest C# version to 3.0.0-beta.20240703.5 (#44865)

* Update Generator Version 3.0.0-beta.20240703.5

* Update SDK codes de_he_2

* Update SDK codes he_me_3

* Update SDK codes ad_co_0

* Update SDK codes st_wo_6

* Update SDK codes mi_pu_4

* Update SDK codes pu_sq_5

* Update SDK codes co_de_1

* update identity version

---------

Co-authored-by: Arcturus Zhang <dapzhang@microsoft.com>

dantedallag committed:
Deployment stacks sdk introduction (#44405)

guptaayush007 committed:
[SQL]  Generating the .NET Sdk for adding manual cutover and OperationPhaseDetails parameter In forward Migration v2 workflow for Migrating Azure sterling Db to azure Hyperscale  DB (#44470)

martincostello committed:
Avoid ServiceBusSenderOptions allocations (#44856)

- Avoid allocating empty `ServiceBusSenderOptions` instances when creating instances of `ServiceBusSender`.
- Fix tests that fail if the machine they are run on does not use US date formatting.

azure-sdk committed:
Increment package version after release of Azure.ResourceManager.Resources (#44871)

azure-sdk committed:
Increment package version after release of Azure.ResourceManager.Sql (#44872)

azure-sdk committed:
Increment package version after release of Azure.ResourceManager.OracleDatabase (#44867)

raych1 committed:
Update GenerateAndBuildLib.ps1 (#44874)

Don't set 'installInstructions' when it's null, otherwise it would fail the schema validation of 'generateOutput'.

welovej committed:
[Api Management] Bump up api version to latest stable 2022-08-01 (#44074)

HarveyLink committed:
[KeyVault] Bump API version to 2023-07-01 (#44690)

azure-sdk committed:
Increment package version after release of Azure.ResourceManager.ApiManagement (#44876)

azure-sdk committed:
Increment package version after release of Azure.ResourceManager.KeyVault (#44877)

azure-sdk committed:
Delete eng/common/InterdependencyGraph.html which is unused. (#44860)

Co-authored-by: Wes Haggard <weshaggard@users.noreply.github.com>

joseharriaga committed:
[Azure OpenAI] Pin OpenAI version (#44708)

Pin OpenAI version, which is used by Azure OpenAI.

christothes committed:
MSAL logging levels now correlate with the `AzureEventSourceListener` configured log level (#44631)

hamshavathimunibyraiah committed:
Creating SDT client with endpoint only and making source input options public (#44862)

* SouceInput options are made public. Tests are updated to access stoarge accounts using managed identity and no loger through SAS urls. Single document translation client can be created using endpoint alone, mainly for SDK to work against containers.

* Updating commit ID of specs repo to the latest

* Fixing build and analyze failures

* Fixing build errors

* Fixing tests failing in MacOS by updating Default credentials with Token credentials while creating BlobServiceClient

* Test recordings validated and updated

azure-sdk committed:
Update AutoRest C# version to 3.0.0-beta.20240707.1 (#44888)

* Update Generator Version 3.0.0-beta.20240707.1

* Update SDK codes co_de_1

christothes committed:
Fix for BearerTokenAuthenticationPolicy hang (#44882)

benbp committed:
Skip project reference tests for openai (#44896)

azure-sdk committed:
Update GitHubEventProcessor version to 1.0.0-dev.20240708.1 (#44898)

Co-authored-by: James Suplizio <jasupliz@microsoft.com>

JimSuplizio committed:
Make @Azure/azure-sdk-eng the owner of .github/workflows (#44900)

azure-sdk committed:
Increment package version after release of Azure.AI.Translation.Document (#44895)

azure-sdk committed:
Sync eng/common directory with azure-sdk-tools for PR 8558 (#44863)

* Support storage network access and worm removal in remove test resources script

* Move storage network access script to common resource helpers file

* Improve storage container deletion resilience

* Plumb through pool variable to live test cleanup template

* Add sleep for network rule application

---------

Co-authored-by: Ben Broderick Phillips <bebroder@microsoft.com>

annelo-msft committed:
ClientModel: Updates to paging types to support sub-client enumerator pattern (#44825)

* Initial checkin of changes to paging APIs

* nits

* rename ToValueCollection method to GetAllValues

* nits

* Add back simplified PageToken; otherwise implementations must always deserialize before using

* rename to ClientToken for general use with LRO rehydration as well

* nits

* make ClientToken non-abstract

* nits

* proposed renames

* backup idea; I'm wondering if RequestOptions can always be handled by the closure

* remove RequestOptions from signatures

* nits

* GetPageCore - enable an overload that doesn't take a page token

* Change GetPage with no parameters to GetFirstPage

* rename ClientToken to ContinuationToken and remove GetFirstPage method

* some updates based on feedback from Krzysztof

* add Async suffix

* move to CurrentPageToken instead of FirstPageToken

* simplified page collection

* tidy

* tidy

* implement interface explicitly

* reimplement GetAllValues to account for possibility GetCurrentPage was called first

* add internal emitted files that generated code will use

* Add stubs for mock paging client

* flush out client a bit

* updates

* Backup of mock client WIP

* rework/rename and add tests prior to implementation

* make first test pass

* sync tests all pass

* backup sync async test base paging tests idea

* separate unit and scenario tests

* clean up sync unit tests

* add async unit tests

* Add GetCurrentPageCore so that GetCurrentPage doesn't advance enumerator

* complete sync scenario tests

* Add async tests

* nits

* Add refdocs

* address pr feedback on clarifying refdoc comments

* update OAI Test csproj to opt out of project reference pipeline

* add more ExcludeFromProjectReferenceToConversion

* add more ExcludeFromProjectReferenceToConversion

* move build variables

* temporarily opt-out Azure.Core.TestFramework

* revert TestFramework change and do it for Azure.Core instead

* try it without OAI csproj mods

martincostello committed:
Avoid ServiceBus*Options allocations (#44885)

- Avoid allocating empty `ServiceBusClientOptions` and `ServiceBusProcessorOptions` instances when creating instances of `ServiceBusClient` and `ServiceBusProcessor`.
- Remove unused `ServiceBusSenderOptions.Clone()` method.

m-redding committed:
[Schema Registry] Generating from TypeSpec for JSON GA (#44881)

* regenerating

* API

* cleanup files

* test recordings

* clean up Avro package

* cleaned up too much...

* fix test recordings

m-redding committed:
[Azure.Core] GeoJsonConverter serialization fix (#44835)

* GeoJsonConverter serialization fix

* formatting

* formatting

* Update GeoJsonConverter.cs

* Update CHANGELOG.md

* add comment

weshaggard committed:
Increment Microsoft.Azure.Functions.Worker.Extensions.WebPubSub version (#44880)

Also update version increment script to error on missing version
property.

hallipr committed:
Use eng/common template for create-tags-and-git-release (#44921)

* Use eng/common template for create-tags-and-git-release

GuptaVertika committed:
Package-2024-01-01 (#44868)

* Package-2024-01-01

* script ran

* workaround of breaking changes

- fix breaking change in GetExpressRouteCrossConnections
- fix breaking change in InboundSecurityRule
- add backward compatibility of `FirewallPacketCaptureContent`
- update api interface

* fix change log version check error

* fix release date

* remove [Obsolete] and fix extension implementation error

---------

Co-authored-by: Mingzhe Huang (from Dev Box) <mingzhehuang@microsoft.com>

christothes committed:
Turn off WarningsAsErrors until we bump System.Text.Json (#44931)

trrwilson committed:
address null purpose check in FileClient GetFiles (#44923)

Commit:
Fix typo in TestEnvironment variable SYSTEM_ACCESSTOKEN (#44933)

scottaddie committed:
Update CHANGELOG entry for Assistants package (#44918)

yogeshmo committed:
[Communication][Email] Updating CHANGELOG for version 1.1.0-beta.1 (#44934)

* [Communication][Email] Updating CHANGELOG for version 1.2.0-beta.1

* Changing version to 1.1.0-beta.1

azure-sdk committed:
add the ability to override default succeeded() conditioning by parameter (#44937)

Co-authored-by: Scott Beddall <scbedd@microsoft.com>

koderjoker committed:
Azure Redis .Net SDK update for new api v2024-03-01 (#44847)

* Update autorest and generated code

* Update ArmRedisModelFactory customization code and generate API file

* Add tests for disable access keys auth

* Update changelog

* Push recordings

* Change csproj version

* Update changelog

* Update arm model factory customization

* Update arm model factory customization code with named arguments

* Update release date in CHANGELOG

---------

Co-authored-by: Kanchan Joshi <kanchanjoshi@microsoft.com>

azure-sdk committed:
Update AutoRest C# version to 3.0.0-beta.20240710.2 (#44947)

* Update Generator Version 3.0.0-beta.20240710.2

* Update SDK codes de_he_2

* Update SDK codes st_wo_6

* Update SDK codes mi_pu_4

* Update SDK codes he_me_3

* Update SDK codes co_de_1

* try to fix spec

---------

Co-authored-by: tadelesh <tadelesh.shi@live.cn>

ArcturusZhang committed:
refine a new api in redis (#44948)

* refine a new api in redis

* fix test cases

archerzz committed:
fix(docs): wrong <param> location in `System.ClientModel` (#44943)

* fix(docs): wrong <param> location in `System.ClientModel`

* Update parameter description to make the meaning more clear.

Co-authored-by: Jesse Squire <jesse.squire@gmail.com>

---------

Co-authored-by: Jesse Squire <jesse.squire@gmail.com>

annelo-msft committed:
Preparation for release of Azure.Core 1.41.0 (#44938)

azure-sdk committed:
Update AutoRest C# version to 3.0.0-beta.20240711.1 (#44953)

annelo-msft committed:
Preparation for release of System.ClientModel v1.1.0-beta.5 (#44939)

* Preparation for release of System.ClientModel v1.1.0-beta.5

* updates per pr fb

azure-sdk committed:
Support regex/negative regex filters for stress test discovery. Add storage env defaults (#44936)

Co-authored-by: Ben Broderick Phillips <bebroder@microsoft.com>

annelo-msft committed:
remove broken links from Core changelogs to address link checker issue (#44958)

jsquire committed:
[Service Bus] Enforce batch limits (#44917)

* [Service Bus] Enforce batch limits

The focus of these changes is to add client-side
logic to force the maximum size for batches when
large message support is enabled and to enforce the
undocumented 4,500 message limit for a single batch.

azure-sdk committed:
Update azure-sdk-build-tools Repository Resource Refs in Yaml files (#44859)

azure-sdk committed:
Increment version for core releases (#44964)

* Increment package version after release of System.ClientModel

* Increment package version after release of Azure.Core

tejasm-microsoft committed:
adds myself to code owners and change test endpoint

tejasm-microsoft committed:
adds myself to code owners and change test endpoint

tejasm-microsoft commented on NuGet.Config:
```diff
@@ -5,6 +5,7 @@
     <!-- Do not add any additional feeds if new packages are needed they need to come from nuget.org or our azure-sdk-for-net DevOps feed -->
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
     <add key="azure-sdk-for-net" value="https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-for-net/nuget/v3/index.json" />
+    <add key="Group Quota Nuget" value="C:\GroupQuotaNuget" />
```
Removed this extra package

tejasm-microsoft commented:
> @tejasm-microsoft :
> 
> Your account lacks the required public GitHub organizations and permissions required of an internal contributor. Please review the Azure SDK onboarding documentation and use the associated Teams channel for support.
> 
> * [Azure SDK onboarding](https://eng.ms/docs/products/azure-developer-experience/onboard/requirements) _(Microsoft internal)_
> * [Azure SDK onboarding assistance](https://teams.microsoft.com/l/channel/19%3ade3fce22c2994be18cac50502c55f717%40thread.skype/Azure%2520SDK%2520Onboarding?groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47) _(Microsoft internal)_
> 
> You can verify the state of your account by running the [Validate-AzsdkCodeOwner](https://github.com/Azure/azure-sdk-tools/blob/main/tools/github/scripts/Validate-AzsdkCodeOwner.ps1) script from the Azure SDK tools repository.
> 
> Please also be sure to add yourself to [CODEOWNERS](https://github.com/Azure/azure-sdk-for-net/blob/main/.github/CODEOWNERS) for this library, if you will be maintaining it going forward.

Added myself to the code owners file, not sure if I followed the right format

Also there seems to be a discrepancy between the output of the Validate-AzsdkCodeOwner, since it seems I am apart of the two required orgs on the github portal?
<img width="799" alt="image" src="https://github.com/user-attachments/assets/86eb6c28-d5ff-4724-a237-ef81fbb55fd6">

seanmcc-msft committed:
Removed Entra App Secrets from Storage tests (#44427)

jsquire committed:
[KeyVault] Sample depedency versions (#44919)

* [KeyVault] Sample depedency versions

The focus of these changes is to bump the versions used by the sample.  The current Identity reference is flagged for vulnerabilities.

* Update sdk/keyvault/samples/getcert/getcert.csproj

azure-sdk committed:
Increment package version after release of Azure.Communication.Email (#44957)

jsquire commented:
> Also there seems to be a discrepancy between the output of the Validate-AzsdkCodeOwner, since it seems I am apart of the two required orgs on the github portal?

@tejasm-microsoft :  Your organizations are not marked as public.   You see them, as to other members of those orgs but they are private on your account.   Please follow the onboarding guidance.

jsquire commented on .github/CODEOWNERS:
```diff
@@ -968,6 +968,9 @@
 # PRLabel: %Spring App Discovery
 /sdk/springappdiscovery/Azure.ResourceManager.*/                   @sunkun99 @archerzz @ArcturusZhang @ArthurMa1978
 
+#PRLabel: %Quota
+/sdk/quota/Azure.ResourceManager.*/                                @tejasm-microsoft
```
```suggestion
# PRLabel: %Quota
/sdk/quota/Azure.ResourceManager.*/                                @tejasm-microsoft

# ServiceLabel: %Quota %Mgmt
# ServiceOwners:                                                   @tejasm-microsoft
```

These need to follow the same format as the other management libraries.   Please move this block - with my suggestion - up to ~L959 to preserve alphabetical ordering by label.

tejasm-microsoft replied:
Followed the order suggested and was able to verify via PS command:
<img width="850" alt="image" src="https://github.com/user-attachments/assets/9f6caf9b-d888-4c93-86f7-78e4d357b419">

jsquire committed:
[EngSys] Clean up local debug code (#44978)

The focus of these changes is to remove what appears to be local debug code that should not have made it into `main`.

jsquire committed:
[KeyVault] More sample dependency versions (#44920)

* [KeyVault] More sample dependency versions

The focus of these changes is to bump dependency versions used in the Share Link sample, moving to a non-vulnerable Identity version.  Because Core is a transitive dependency, removed the explicit reference.

* Update sdk/keyvault/samples/sharelink/ShareLink.csproj

annelo-msft committed:
Update Core package version for Azure.Core 1.41.0 release (#44980)

tejasm-microsoft committed:
edit codeowners file

tejasm-microsoft committed:
edit codeowners file

JoshLove-msft committed:
Prepare for release (#44982)

minwoolee-msft committed:
Disabling all CallingServer test as it is in the process of decomissioning (#44979)

JimSuplizio committed:
Adding some verbose output to help diagnose something going awry with docs gen (#44965)

* Adding some verbose output to help diagnose something going awry with docs gen

* see what happens if there is no target

* update commands that fail to log warnings

jsquire committed:
[EngSys] Remove baseline error (#44985)

The focus of these changes is to remove a baseline error suppression for an account that is no longer an active Microsoft contributor.

jsquire committed:
[CODEOWNERS] Remove invalid account (#44984)

Removing a legacy account that the linter is now flagging as invalid.

azure-sdk committed:
Sync eng/common directory with azure-sdk-tools for PR 8598 (#44935)

* Set storage account test resources to disable blob public access

* Skip adding network rules to storage accounts that don't need them during cleanup

* Add succeeded check to set pipeline subnet info step

* Disable network firewall by default in resource creation/removal

---------

Co-authored-by: Ben Broderick Phillips <bebroder@microsoft.com>

johncrim committed:
fix(otel logging): Include ILogger CategoryName in logger output (#44754)

* fix(telemetry transmission): Include ILogger CategoryName in logger output

Previously this field was omitted, which was not consistent with normal ILogger behavior, or with Application Insights classic behavior.

fixes: #44746

* improve(otel exporter tests): Verify that logger CategoryName is in Exception log entries

* improve(live metrics): Include LogRecord.CategoryName in live metrics

It's included in live metrics using app insights classic, so should be included here too.

* docs(changelog): Update changelog with "add CategoryName" info

Commit:
prep exporter release 1.4.0-beta1 (#44981)

JoshLove-msft committed:
Fix EventGrid Namespaces readme (#44992)

subhra231 committed:
update Versions of the packages (#44988)

vipray-ms committed:
Updating dotnet sdk for NewRelic (#44852)

azure-sdk committed:
Increment package version after release of Azure.ResourceManager.Redis (#44997)

azure-sdk committed:
Increment package version after release of Azure.ResourceManager.NewRelicObservability (#45001)

tejasm-microsoft committed:
resolves merge conflict

azure-sdk commented:
**API change check**

APIView has identified API level changes in this PR and created following API reviews.

[Azure.ResourceManager.Quota](https://apiview.dev/Assemblies/Review/9eae2f6971e14eb38ccddc391596f535?revisionId=f5bbf62f89a6487aa272821a6508cba2)

jsquire committed:
[CODEOWNERS] Removing Pallavi's ownership (#45006)

The focus of these changes is to remove ownership areas for Pallavi, as their account is no longer associated with Microsoft.

amnguye committed:
[Storage] Fix for truncating URL paths with "#" (#44941)

* WIP - still need to record tests

* Recordings

* Cleanup

* Fixed recordings

* Added fix for Shares and Datalake, Updated tests to use OAuth, Updated Changelogs

* Rerecord files tests

* Merge and record tests

JonathanCrd committed:
[Azure.Data.Tables] DeleteEntity overload (#44986)

* overload method

* Update API

* Update samples

* Update changelog

* Fix error in parameter description

m-redding committed:
[Schema Registry] Default SchemaFormat to the Content-Type header value (#45007)

* fixes

* fixes

* clean up client

JimSuplizio committed:
Add Update Remove-Item command line options (#45010)

Commit:
update integration test with CategoryName (#44989)

tejasm-microsoft commented:
Added myself to CODEOWNERS file and verified sufficient permissions via Powershell commands:
<img width="749" alt="image" src="https://github.com/user-attachments/assets/f2b6a7ab-4899-431f-930e-8431f7510068">
@jsquire

seanmcc-msft committed:
Reenabled live Audience tests (#44972)

JoshLove-msft committed:
Cncf namespace samples (#44994)

* Add CNCF sample

* Update readme

* new recordings

* event

* recording

* Use dynamic

amnguye committed:
[Storage] [WebJobs] Added test for Blob Path for various blob names to check for blob name validation (#45015)

* Added test for Blob Path for possible blob names, to ensure paths are not truncated

* CI changelog error

* Fix CI change log part 2

tejasm-microsoft committed:
removes unneccesary prepended slash for GQ serialization

tejasm-microsoft committed:
removes unneccesary prepended slash for GQ serialization

azure-sdk commented:
**API change check**

APIView has identified API level changes in this PR and created following API reviews.

[Azure.ResourceManager.Quota](https://apiview.dev/Assemblies/Review/9eae2f6971e14eb38ccddc391596f535?revisionId=f5bbf62f89a6487aa272821a6508cba2)

trrwilson committed:
Azure OpenAI: sample/snippet update (request options for On Your Data) (#44956)

* Update README.md

The 'options' is needed to use own data and  Search engine:

ChatCompletion completion = chatClient.CompleteChat(
    [
        new UserChatMessage("How to troubleshoot high cpu at user pool level"),
    ], options);

* sample->snippet update

---------

Co-authored-by: Liwei Yin <43456062+Liwei-Yin@users.noreply.github.com>

seanmcc-msft committed:
Storage STG 94 GA release (#45018)

live1206 commented:
- The tests are failing [here](https://dev.azure.com/azure-sdk/public/_build/results?buildId=3963424&view=ms.vss-test-web.build-test-results-tab&runId=48830982&resultId=103974&paneView=debug), please use test-proxy to upload the recordings, [details](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/resourcemanager/Azure.ResourceManager/docs/TestGuide.md#test-proxy).
> Generated code is not up to date.
    You may need to rebase on the latest main, 
    run 'eng\scripts\Update-Snippets.ps1' if you modified sample snippets or other *.md files (https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#updating-sample-snippets), 
    run 'eng\scripts\Export-API.ps1' if you changed public APIs (https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#public-api-additions). 
    run 'dotnet build /t:GenerateCode' to update the generated code and samples.
- CI is failing as above, please make sure you merge the latest main and regen SDK and export the API

azure-sdk committed:
Update AutoRest C# version to 3.0.0-beta.20240715.1 (#45020)

* Update Generator Version 3.0.0-beta.20240715.1

* Update SDK codes co_de_1

* remove unused configuration.json and tspCodeModel.json

---------

Co-authored-by: FAREAST\chunyu <chunyu@microsoft.com>

azure-sdk committed:
Increment package version after release of Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents (#45011)

azure-sdk committed:
Update AutoRest C# version to 3.0.0-beta.20240716.2 (#45027)

* Update Generator Version 3.0.0-beta.20240716.2

* Update SDK codes de_he_2

* Update SDK codes he_me_3

* Update SDK codes st_wo_6

* Update SDK codes ad_co_0

* Update SDK codes mi_pu_4

* Update SDK codes pu_sq_5

* fix compilation error due to parameter name changes

---------

Co-authored-by: Mingzhe Huang (from Dev Box) <mingzhehuang@microsoft.com>

JimSuplizio committed:
Temporarily disable MSDocs publishing for Azure.Messaging.EventGrid.SystemEvents (#45033)

Commit:
[AzureMonitor] add links to docs in the Readme (#45017)

* add links to docs in the Readme

* pr feedback

christothes committed:
Remove duplicate SYSTEM_ACCESSTOKEN (#45004)

alrod committed:
Fixing target base scale instance concurrency for queues (#44661)

* Fixing target base scale instance concurrency for queues

* New commit

* Adding changelog notes

* Fix

---------

Co-authored-by: Amanda Nguyen <amnguye@microsoft.com>

memontic-ms committed:
[Azure.Communication.Messages] Update README (#44987)

* Update Azure.Communication.Messages README

azure-sdk committed:
Increment package version after release of Azure.Monitor.OpenTelemetry.Exporter (#45037)

azure-sdk committed:
Increment version for storage releases (#45039)

weshaggard committed:
Update Preview Test Sub to use new connection (#45035)

In order to get both deployment and clean-up working we need
to have a connection that points at the right subscription. Today azure-sdk-tests points to the name Test Sub but I created azure-sdk-tests-preview connection which points to the Preview Test Sub.

This will fix the clean-up step which is currently failing for things deployed in Preview.

amnguye committed:
Bumped Storage, Blobs, and Queues package dependency version (#45016)

amnguye committed:
[Storage] [DataMovement] Improve checking on part completion for pause/stop/completion status (#45038)

* WIP

* Make CI happy

* Revert "Make CI happy"

This reverts commit a7e80cfc27850b62075ea05c22475080ed79b7f4.

* WIP

* Cleanup

* More cleanup

amnguye committed:
Updated Changelog for DataMovement July Release (#45040)

Commit:
[AzureMonitorDistro] prep distro 1.3.0-beta.1 (#44991)

* prep distro 1.3.0-beta.1

* change to package reference

amnguye committed:
Changed DataMovement Blobs and File Shares to use package dependency, Added File Shares to packages.data.props (#45042)

azure-sdk committed:
Increment package version after release of Azure.Monitor.OpenTelemetry.AspNetCore (#45043)

azure-sdk committed:
Increment version for storage releases (#45045)

tejasm-microsoft committed:
addresses PR comments. Fixes SubscriptionPUT/PATCH and adds assets

tejasm-microsoft committed:
addresses PR comments. Fixes SubscriptionPUT/PATCH and adds assets

tejasm-microsoft commented:
> * The tests are failing [here](https://dev.azure.com/azure-sdk/public/_build/results?buildId=3963424&view=ms.vss-test-web.build-test-results-tab&runId=48830982&resultId=103974&paneView=debug), please use test-proxy to upload the recordings, [details](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/resourcemanager/Azure.ResourceManager/docs/TestGuide.md#test-proxy).
> 
> > Generated code is not up to date.
> > You may need to rebase on the latest main,
> > run 'eng\scripts\Update-Snippets.ps1' if you modified sample snippets or other *.md files (https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#updating-sample-snippets),
> > run 'eng\scripts\Export-API.ps1' if you changed public APIs (https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#public-api-additions).
> > run 'dotnet build /t:GenerateCode' to update the generated code and samples.
> 
> * CI is failing as above, please make sure you merge the latest main and regen SDK and export the API

Resolved the failing tests  and added the assets.json via test proxy:
https://github.com/Azure/azure-sdk-assets/tree/net/quota/Azure.ResourceManager.Quota_09eadc6fbb/net/sdk/quota/Azure.ResourceManager.Quota/tests/SessionRecords/GroupQuotaTests

Also ran Code Checks script locally and came back with no errors

azure-sdk commented:
**API change check**

APIView has identified API level changes in this PR and created following API reviews.

[Azure.ResourceManager.Quota](https://apiview.dev/Assemblies/Review/9eae2f6971e14eb38ccddc391596f535?revisionId=f5bbf62f89a6487aa272821a6508cba2)

live1206 committed:
Merge branch 'main' into users/tejasma/generateSdkQuota

live1206 committed:
merge latest main and regen

live1206 committed:
merge latest main and regen

live1206 commented:
> > > Generated code is not up to date.
> > > You may need to rebase on the latest main,
> > > run 'eng\scripts\Update-Snippets.ps1' if you modified sample snippets or other *.md files (https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#updating-sample-snippets),
> > > run 'eng\scripts\Export-API.ps1' if you changed public APIs (https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#public-api-additions).
> > > run 'dotnet build /t:GenerateCode' to update the generated code and samples.
> > 
> > 
> > 
> > * CI is failing as above, please make sure you merge the latest main and regen SDK and export the API
> 
> Resolved the failing tests and added the assets.json via test proxy: https://github.com/Azure/azure-sdk-assets/tree/net/quota/Azure.ResourceManager.Quota_09eadc6fbb/net/sdk/quota/Azure.ResourceManager.Quota/tests/SessionRecords/GroupQuotaTests
> 
> Also ran Code Checks script locally and came back with no errors

I merged the latest main to your branch and re-generate the SDK for you, the corresponding CI should be fixed now.

The tests are still failing, you can reproduce the failing tests by running the tests locally.
- Remove the local recording located in `\src\azure-sdk-for-net\.assets`
- Open a cmd under `\src\azure-sdk-for-net\sdk\quota\Azure.ResourceManager.Quota`, restore the remote recordings by running "test-proxy restore -a ./assets.json"
- run the tests, you should see the test failure

azure-sdk commented:
**API change check**

APIView has identified API level changes in this PR and created following API reviews.

[Azure.ResourceManager.Quota](https://apiview.dev/Assemblies/Review/9eae2f6971e14eb38ccddc391596f535?revisionId=f5bbf62f89a6487aa272821a6508cba2)

amnguye committed:
[Storage] [Webjobs Extension] Updated changelog to prepare for WebJobs Storage Extension release (#45041)

azure-sdk committed:
Increment version for storage releases (#45054)

subhra231 committed:
Update Changelog and readme files to include --prerelease (#45047)

* Update Changelog File to include release date.

* Update Readme to add --prerelease option

azure-sdk committed:
Increment package version after release of Azure.Messaging.EventGrid (#45056)

ShivangiReja committed:
[Search] GA Features for API Version 2024-07-01 (#44485)

* [Search] GA Features for API Version 2024-07-01

Commit:
[AzureMonitor] add Log Scopes example to Readme (#44749)

* add Log Scopes example to Readme

* pr feedback

* add AzureMonitor to example code

* pr feedback

* Update sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/README.md

Co-authored-by: Cijo Thomas <cithomas@microsoft.com>

* add another test

* pr feedback

* pr feedback

* Update sdk/monitor/Azure.Monitor.OpenTelemetry.AspNetCore/README.md

Co-authored-by: Rajkumar Rangaraj <rajrang@microsoft.com>

* fix

* pr feedback

---------

Co-authored-by: Cijo Thomas <cithomas@microsoft.com>
Co-authored-by: Rajkumar Rangaraj <rajrang@microsoft.com>

ArthurMa1978 committed:
Prepare release for mysql 1.1.0 (#45051)

azure-sdk committed:
Increment package version after release of Azure.ResourceManager.MySql (#45067)

azure-sdk committed:
Increment package version after release of Azure.Search.Documents (#45062)

HarveyLink committed:
[AppService] Bump api version to 2023-12-01 (#43706)

m-redding committed:
[Azure.Messaging.EventHubs] Prepare for release (#45061)

* [Event Hubs] Prepare for release

* update version number to be greater than the beta

* update version

scottaddie committed:
Remove duplicate credential instantiation line in ACS Email README (#45076)

jsquire committed:
[Service Bus] July 2024 release prep (#45075)

* Preparation for the July 2024 release.

JimSuplizio committed:
Nightly docs publish should honor skipPublishDocMs (#45077)

azure-sdk committed:
Increment package version after release of Azure.Messaging.EventHubs (#45079)

jsquire committed:
[Service Bus] Change log typo (#45082)

The focus of these changes is to fix a typo that collapsed bullet points into one item.

azure-sdk committed:
Sync eng/common directory with azure-sdk-tools for PR 8602 (#45019)

* add additional argument to Save-Package-Properties to allow for usage in pull request context

---------

Co-authored-by: Scott Beddall (from Dev Box) <scbedd@microsoft.com>
Co-authored-by: Scott Beddall <45376673+scbedd@users.noreply.github.com>
Co-authored-by: Wes Haggard <weshaggard@users.noreply.github.com>

m-redding committed:
[Azure.Messaging.EventHubs.Processor] Prepare for release (#45063)

* updates

* version

* updates

JonathanCrd committed:
[Azure.Data.Tables] Update version for release (#45058)

* Update release version in Changelog and project

* Update release date

amnguye committed:
[Storage] [Tests] Fix OAuth Storage Samples (#45083)

* Fix storage samples

* Update snippets

azure-sdk committed:
Increment version for eventhub releases (#45085)

* Increment package version after release of Azure.Messaging.EventHubs.Processor

* Update Azure.Messaging.EventHubs.Processor.csproj

* Update CHANGELOG.md

---------

Co-authored-by: Madalyn Redding <66138537+m-redding@users.noreply.github.com>

scottaddie committed:
Reformat DefaultAzureCredential flow chart (#45086)

rajuanitha88 committed:
Using federated auth for ACS tests (#44886)

* Using federated auth for ACS tests

* Removing usage of secrets for Shortcodes

* Removing PPE and INT Live tests

* Addressing comments to maintain consistency across languages

seanmcc-msft committed:
Updated Storage codeowners (#45088)

ArthurMa1978 committed:
Merge branch 'main' into quota

ArthurMa1978 committed:
update

ArthurMa1978 committed:
update

tejasm-microsoft committed:
Uses the given test client rather than using own

tejasm-microsoft committed:
Uses the given test client rather than using own

tejasm-microsoft committed:
pull in changes from remote

tejasm-microsoft committed:
adds comments and uses new resources

tejasm-microsoft committed:
adds comments and uses new resources

tejasm-microsoft committed:
rebase from main and updated assets.json

tejasm-microsoft committed:
resolve merge conflict

ArthurMa1978 committed:
Merge branch 'main' into quota

ArthurMa1978 committed:
update.

ArthurMa1978 committed:
update

ArthurMa1978 committed:
fix changelog
